### ISSUE TITLE
Issue after upgrade to 1.4.1

### ISSUE THREAD

**Issue by CiquattroFPV** [keatontaylor/alexa_media_player#289]
I'm here for ask an help.. after the upgrade to 1.4.1 it stopped working but from my side seem it's configured correctly.

Anyone can help me to understadn what's wrong?

that's my configuration:

configuration.yaml
```
alexa_media:
  accounts:
    - email: !secret amazonemail
      password: !secret amazonpwd
      url: !secret amazonurl_it  #this is the amazon url for italy  amazon.it
      scan_interval: 30
```

notify.yaml

```
- platform: alexa_media
  name: alexa_media
```

automation.yaml
```
- id: notify-hass-start
  alias: Home Assistant Start - Notify
  initial_state: 'on'
  trigger:
    platform: homeassistant
    event: start
  action:
  - service: notify.hassnico_bot
    data:
      message: "Home Assistant Start at {{ states('sensor.date_time') }}"
  - service: notify.alexa_media
    data:
      target: 
        - media_player.nicolo_s_bose_home_speaker_500
      data:
        type: announce
      message: >
       {{ [
       "Home Assistant è nuovamente disponibile",
       "Home Assistant è operativo",
       "Home assistant is back bitches",
       "Home assistant è avviato",
       ] | random }}
```
that's the debug log.. 
```
2019-08-17 20:25:43 WARNING (SyncWorker_5) [homeassistant.components.efergy.sensor] Could not update status for Energy Usage
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] n***********e@gmail.com: Found 4 devices, 1 bluetooth
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] Locale it-it found for 7****************************4d6
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] DND False found for 7****************************4d6
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] Locale en-us found for f****************************621
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] DND False found for f****************************621
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] Locale it-it found for G************XR4
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] DND False found for G************XR4
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] Locale it-it found for 3****************************b3b
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] DND False found for 3****************************b3b
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] n***********e@gmail.com: Existing: [<Entity Nico Bose Home: standby>, <Entity Salotto: standby>, <Entity Echo Dot di Nico: standby>, <Entity This Device: standby>] New: []; Filtered by: include_devices: [] exclude_devices:[]
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] n***********e@gmail.com: Updated last_called: {'serialNumber': '7****************************4d6', 'timestamp': 1566065968431}
2019-08-17 20:25:53 DEBUG (SyncWorker_4) [custom_components.alexa_media] n***********e@gmail.com: last_called changed: {'serialNumber': '7****************************4d6', 'timestamp': 1566065970269} to {'serialNumber': '7****************************4d6', 'timestamp': 1566065968431}
2019-08-17 20:25:53 DEBUG (SyncWorker_7) [custom_components.alexa_media.media_player] Nico Bose Home is last_called: 7****************************4d6
2019-08-17 20:25:53 DEBUG (SyncWorker_18) [custom_components.alexa_media.media_player] n***********e@gmail.com: Refreshing Salotto
2019-08-17 20:25:53 DEBUG (SyncWorker_18) [custom_components.alexa_media.media_player] Salotto: Last_called check: self: f****************************621 reported: 7****************************4d6
2019-08-17 20:25:54 DEBUG (Thread-4135) [custom_components.alexa_media] n***********e@gmail.com: Websocket created: <WebsocketEchoClient(Thread-4136, started daemon 1869642688)>
2019-08-17 20:25:54 DEBUG (SyncWorker_0) [custom_components.alexa_media.media_player] n***********e@gmail.com: Refreshing Echo Dot di Nico
2019-08-17 20:25:54 DEBUG (SyncWorker_0) [custom_components.alexa_media.media_player] Echo Dot di Nico: Last_called check: self: G************XR4 reported: 7****************************4d6
2019-08-17 20:25:54 DEBUG (SyncWorker_1) [custom_components.alexa_media.media_player] n***********e@gmail.com: Refreshing This Device
2019-08-17 20:25:54 DEBUG (SyncWorker_1) [custom_components.alexa_media.media_player] This Device: Last_called check: self: 3****************************b3b reported: 7****************************4d6
2019-08-17 20:25:55 DEBUG (SyncWorker_19) [custom_components.alexa_media.media_player] n***********e@gmail.com: Refreshing Nico Bose Home
2019-08-17 20:25:55 DEBUG (SyncWorker_19) [custom_components.alexa_media.media_player] Nico Bose Home: Last_called check: self: 7****************************4d6 reported: 7****************************4d6
2019-08-17 20:25:57 DEBUG (Thread-4136) [custom_components.alexa_media] n***********e@gmail.com: Websocket succesfully connected
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Message: Home Assistant è operativo, kwargs: {'target': ['media_player.nicolo_s_bose_home_speaker_500'], 'data': OrderedDict([('type', 'announce')])}
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: media_player.nicolo_s_bose_home_speaker_500 against (<Entity Nico Bose Home: standby>, Nico Bose Home, 7****************************4d6, media_player.nicolo_s_bose_home_speaker_500)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Converting: media_player.nicolo_s_bose_home_speaker_500 to (entities): <Entity Nico Bose Home: standby>
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: media_player.nicolo_s_bose_home_speaker_500 against (<Entity Salotto: standby>, Salotto, f****************************621, media_player.salotto)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: media_player.nicolo_s_bose_home_speaker_500 against (<Entity Echo Dot di Nico: standby>, Echo Dot di Nico, G************XR4, media_player.echo_dot_di_nicolo)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: media_player.nicolo_s_bose_home_speaker_500 against (<Entity This Device: standby>, This Device, 3****************************b3b, media_player.this_device_2)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: <Entity Nico Bose Home: standby> against (<Entity Nico Bose Home: standby>, Nico Bose Home, 7****************************4d6, media_player.nicolo_s_bose_home_speaker_500)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Converting: <Entity Nico Bose Home: standby> to (serialnumbers): yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: <Entity Nico Bose Home: standby> against (<Entity Salotto: standby>, Salotto, f****************************621, media_player.salotto)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: <Entity Nico Bose Home: standby> against (<Entity Echo Dot di Nico: standby>, Echo Dot di Nico, G************XR4, media_player.echo_dot_di_nicolo)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Testing item: <Entity Nico Bose Home: standby> against (<Entity This Device: standby>, This Device, 3****************************b3b, media_player.this_device_2)
2019-08-17 20:30:54 DEBUG (SyncWorker_19) [custom_components.alexa_media.notify] Announce targets: ['7****************************4d6'] entities: [<Entity Nico Bose Home: standby>]
```

**Comment by alandtse** [comment]
This seems the same as #285 since it's a third-party speaker. Please get us the alexapy debug logs and confirm you can create a routine in the Alexa app to send an announcement to the speaker.

**Comment by CiquattroFPV** [comment]
@alandtse yes seems the same issue of the tichet #285 if I change the type: announce to type: tts it work.. maybe can be merged or better to close this issue?

**Comment by alandtse** [comment]
Closing so we can monitor in #285.  Please note a new version is out that changed the architecture so this bug may never be resolved in the 1.4.1 branch.
